lighthouse-core/audits/byte-efficiency/uses-long-cache-ttl.js

```diff
@@ -202,6 +202,12 @@ class CacheHeaders extends Audit {
           cacheControl
         );
 
+        // Ignore assets where policy implies they should not be cached long periods
+        if (cacheControl &&
+          (cacheControl['must-validate'] || cacheControl['no-cache'] || cacheControl['private'])) {
```

Comment:
Can we figure out how to consolidate some of this with the logic in `computeCacheLifetimeInSeconds` in L115? right now these type of checks are in there to return a lifetime of `0`

Reply:
maybe? these cache controls could still have ttls, so it'd make `computeCacheLifetimeInSeconds` a bit of a liar. I get that if we have this method return 0 for all cases we want to ignore, things work. which is cool, I'm just arguing semantics I suppose.

Reply:
it doesn't have to be moving this there, we can extract the L115 part too (and probably should it's not the most obvious right now). I'm just advocating for collecting the special cases we handle so it's clear what's going on.

Reply:
Done. How's that?